style.css

```diff
@@ -0,0 +1,137 @@
+* {
```

Comment:
- [ ] As suggested by the project requirements, I kindly suggest that you create an  `index.css` file